### PR TITLE
Update reg.yaml

### PR DESCRIPTION
**Pull request by DevinCrowley**
Add installation tab, update examples link to new location of demo.ipynb, and rewrite main text (details).

### CODE REVIEW

**Review by falkben — CHANGES_REQUESTED** [review]
Getting a 503 error when I try to access examples.  I think that should now link to: https://ardent.neurodata.io/demo/demo.html right?

**Comment by DevinCrowley** [comment]
That's right

File: content/tools/reg.yaml [hunk 1]
```diff
@@ -5,17 +5,18 @@ name: reg
 
 github: "neurodata/ardent"
 docs: "https://ardent.neurodata.io"
-installation:
-examples: "https://nbviewer.jupyter.org/github/neurodata/ardent/blob/master/demo/demo.ipynb"
+installation: "https://ardent.neurodata.io/install.html"
+examples: "https://ardent.neurodata.io/demo/demo.html"
```
Comment (jovo) [inline content/tools/reg.yaml:9]: this should link here: https://ardent.neurodata.io/tutorial.html

Comment (jovo) [reply content/tools/reg.yaml:9]: i'm resolving conversation to approve, but please update link

**Review by falkben — APPROVED** [review]

**Review by falkben — APPROVED** [review]

**Pull request merged**